### PR TITLE
release 2.2.0

### PR DESCRIPTION
**Pull request by wenchajun**
### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+- Feat: Namespace level CRDs and logging with FluentBit Daemonset (#630)
+
+### ENHANCEMENT
+- Update controller-gen to v0.11.3 & update Makefile CRD_OPTIONS (#624)
+- Fluentd add volumes & volumeClaimTemplates (#633)
+- Make v2 importable (#631)
+- Create or patch rbac objects (#635)
+- Remove the deprecated -i flag for go build (#638)
+- Add charts test (#639)
+- Upgrade fluent-bit to v2.0.10 (#640)
```
Comment (benjaminhuo) [inline CHANGELOG.md:14]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+- Change gelfShortMessgeKey to gelfShortMessageKey (#643)
+- Enable annotations for service account in fluentbit/fluentd (#647)
+- Update README to mention new namespace FluentBit CRDs (#649)
+- Add fluent-operator.drawio (#650)
+- Updated image of operator architecture (#651)
+- Automatic pushing of fluentbit-debug version images (#656)
+- Add service configurations for the components (#657)
+- Make RBAC comptaible with multiple instances (#658)
+- Add the YAML file for edge nodes to the Helm file (#668)
+- Fluent-bit upgrade to v2.0.11 (#684)
```
Comment (benjaminhuo) [inline CHANGELOG.md:26]:
```suggestion
- Upgrade Fluent Bit to v2.0.11 (#684)
```

Comment (wenchajun) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+- Explain how to disable component controllers in the README (#642)
+- Change gelfShortMessgeKey to gelfShortMessageKey (#643)
+- Enable annotations for service account in fluentbit/fluentd (#647)
+- Update README to mention new namespace FluentBit CRDs (#649)
+- Add fluent-operator.drawio (#650)
+- Updated image of operator architecture (#651)
+- Automatic pushing of fluentbit-debug version images (#656)
+- Add service configurations for the components (#657)
+- Make RBAC comptaible with multiple instances (#658)
+- Add the YAML file for edge nodes to the Helm file (#668)
```
Comment (benjaminhuo) [inline CHANGELOG.md:24]: `Add support for collecting edge metrics in Helm chart` should be in the features section

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**